### PR TITLE
PF-3520 - Update secondary menu styling as per new designs

### PR DESCRIPTION
**Pull request by whomwah**
# TODO
- [x] Release notes

screen shot 2017-11-03 at 10 33 06
screen shot 2017-11-03 at 11 16 47
screen shot 2017-11-03 at 11 16 54
screen shot 2017-11-03 at 11 16 59
screen shot 2017-11-03 at 11 17 03

### CODE REVIEW

**Comment by toni-leigh** [comment]
Shouldn't really be linking tickets and screen shots of the app in the open source repo - it doesn't make much sense in a general context to be so specific, plus, it's helpful to think of the changes as generic when improving Carbon

**Comment by whomwah** [comment]
@toni-leigh could you recommend an approach in order to point the relevant people to the correct content changes?

**Comment by andrewjtait** [comment]
@whomwah @toni-leigh ticket numbers are an ok reference, as long as there is more to the title and description to explain the ticket. I'd avoid links to our JIRA board or designs etc though.

The screenshots you have here are ok as they aren't actually of the app 👍

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 * DropdownFilter now uses a stylized Link component for the Create button.
 * DropdownFilter now has `createText` & `createIconType` props for customizable create button text and create button Icons (limited to current Carbon Icon types).
+* Update `secondry` Menu styling as per new designs
```
Comment (andrewjtait) [inline CHANGELOG.md]: 2.2.2 is already released - id target this at 2.3.0

**Comment by toni-leigh** [comment]
@whomwah 

I actually like raising issues here - I see more than one benefit:

1. Everything is visible, problem, solution, PR etc. and related discussions. Someone who doesn't have access to Jira can see everything
2. It gets me to think about the ticket problem in a generic sense meaning what I provide is more likely to be useful.
3. It provides an ID to link up commits etc. like using the ticket code in Jira.
4. It leaves no proprietary information in the repo, I feel in some circumstances, particularly around security issues, even the hint that we're working on something for our app could provide an astute hacker with a pointer towards other potential problems (long shot I know)

Not essential, but worth a thought as a process when working open source.

**Comment by toni-leigh** [comment]
Also, if you raise an issue in enough time, someone else might do the work for you!

**Comment by whomwah** [comment]
@andrewjtait is your change request (https://github.com/Sage/carbon/pull/1612#pullrequestreview-73810980) covered by 742be1d

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,12 @@
+# 2.3.0
+
+## CSS Changes
+
+* `Menu` has been updated based on slightly new designs. `secondary` menus now support `Icon` components as children.
+* `Portrait` size `extra-small` has reduced from `26px` to `25px`.
+* `Portrait` size `medium-small` has reduced from `50px` to `40px`.
```
Comment (SeanArmstrong) [inline CHANGELOG.md]: This seems like quite a large change

**Comment by ianrobsonsage** [comment]
QA Passed

Loaded Carbon demo site locally on branch PF-3520

Tested Menu component in Chrome, Firefox, Safari, IE and Edge. No issues found. Tested both Primary and Secondary styles.

Compared with how current version of Menu component of live Carbon site. No issues found.

**Pull request merged**